src/test/java/ibm/jceplus/junit/openjceplusfips/integration/TestAll.java

```diff
@@ -7,10 +7,10 @@
  */
 package ibm.jceplus.junit.openjceplusfips.integration;
 
-import org.junit.platform.suite.api.SelectClasses;
+import org.junit.platform.suite.api.SelectPackages;
 import org.junit.platform.suite.api.Suite;
 
+@SelectPackages({"ibm.jceplus.junit.openjceplusfips.integration"})
 @Suite
-@SelectClasses({TestTLS.class})
 public class TestAll {
-}
+}
```

Comment:
Add new line.

Reply:
Added a new line between @SelectPackages and @Suite and new line at the end of the class.